docs/how-to-guides/09_how-to-get-table-information.md

```diff
@@ -88,3 +88,66 @@ In the example shown below, the `contract` smart contract's table `profiles` is
   }));
 })();
 ```
+
+## Query Data using the KV API 
+The KV API is a new api which allows smart contract developers to create datastore key value tables on-chain. KV tables can have muliple indexes, a single unique index and many non-unique indexes. If the smart contract uses KV tables use the get_kv_table_rows RPC call to query data.
```

Comment:
a kv table can have multiple unique indexes as well. I think the way it is phrased now it leaves the impression it can have only one unique index.
there is a restriction though, a kv table must have at least one unique index.

Reply:
Clarify indexes

Reply:
Clarified index use